module/Photo/view/partial/tags.phtml

```diff
@@ -7,8 +7,10 @@ $this->scriptUrl()->requireUrl('photo/photo/tag/remove', ['photo_id', 'lidnr'])
     $tagCount = $this->photo->getTags()->count();
     $i = 1;
     ?>
-    <?= (0 !== $tagCount) ? $this->translate('In this photo:') : '' ?>
-    <div class="tagList tagList-<?= $this->photo->getId() ?>">
+    <?php if (0 !== $tagCount): ?>
+        <span class="tag-title"><?= $this->translate('In this photo:') ?></span>
+    <?php endif; ?>
```

Comment:
Would it be an idea to add an alternate text in case no tags are present? Something like `No one has been tagged in this photo yet. Tag someone now!`

Reply:
Good idea, we just have to make sure we do not `remove()` the element, as we cannot add translations in the JS file which handles the removal of tags.

I'll add it when I have some time this weekend.